controller/common/node_info.py

```diff
@@ -11,27 +11,35 @@ def __init__(self, node_id):
         node_name, nvme_nqn, fc_wwns_str, iscsi_iqn = utils.get_node_id_info(node_id)
         fc_wwns = fc_wwns_str.split(config.PARAMETERS_FC_WWN_DELIMITER)
```

Comment:
I think we should either use `string_to_array` (which IMO should be called `str_to_list`) here in addition to the [mediator](https://github.com/IBM/ibm-block-csi-driver/blob/ba300e783c8a89f427845f87c0a6c5e3ff44bb78/controller/array_action/array_mediator_xiv.py#L378)s, or only in the `Initiators` class, but no need to do similar logic more than once

Reply:
CSI-4250

Reply:
is it related to this PR?

Reply:
yes, because this PR adds both new `_filter_empty_parts` and more [`string_to_array`](https://github.com/IBM/ibm-block-csi-driver/blob/543899a6915750d5f4d3a6be93c5b891a6e61e02/controller/array_action/array_mediator_xiv.py#L377) calls, which share duplicated `strip()` logic

Reply:
`string_to_array` doing `split()`, I don't think that `Initiators` should do it too, how do you suggest removing `string_to_array` but still splitting wherever needed?

Reply:
if `split()` belongs outside `Initiators`, that doesn't mean we need to keep `string_to_array` in the project.
we can just split instead of calling `string_to_array`, and leave the `strip()` part to `Initiators`

Reply:
done